src/events/index.test.ts

```diff
@@ -51,7 +49,7 @@ describe('allEvents', () => {
 		// Be sure to clear all the auto-added event handlers first, or else they'll mess up our count.
 		// Casting a read-only list into a regular list is bad practice, but this is for testing purposes.
 		// Don't do this at home.
-		(allEventHandlers as Map<keyof ClientEvents, EventHandler>).clear();
+		(allEventHandlers as Map<string, unknown>).clear();
```

Comment:
Why unknown?

Reply:
Since we only wanna clear the Map, we don't care much about what's in it. This method saves us a type import